packages/beacon-node/src/execution/engine/http.ts

```diff
@@ -139,9 +158,19 @@ export class ExecutionEngineHttp implements IExecutionEngine {
       this.updateEngineState(getExecutionEngineState({payloadError: error, oldState: this.state}));
     });
 
-    this.rpc.emitter.on(JsonRpcHttpClientEvent.RESPONSE, () => {
-      this.updateEngineState(getExecutionEngineState({targetState: ExecutionEngineState.ONLINE, oldState: this.state}));
-    });
+    if (!opts?.disableClientVersionFetch) {
+      this.rpc.emitter.on(JsonRpcHttpClientEvent.RESPONSE, () => {
+        if (this.clientVersion === undefined) {
+          // This statement should only be called first time receiving response after start up
+          this.getClientVersion(getLodestarClientVersion(this.opts)).catch((e) => {
+            this.logger.error("Unable to get execution client version", {}, e);
```

Comment:
We might wanna log this as a `warn` as it does not require user intervention and there is nothing the user can do, besides maybe updating their EL.

Do you know by chance if all ELs support this already? otherwise we might even wanna go with `debug`

Reply:
@nflaig No, Besu doesn't support it yet. Will change it to `debug`